types/chart.js/index.d.ts

```diff
@@ -97,9 +97,20 @@ declare class Chart {
         [key: string]: Chart;
     };
 }
+type Plugin = Chart.PluginServiceGlobalRegistration & Chart.PluginServiceRegistrationOptions;
+interface PluginDescriptor {
+    plugins: Plugin;
```

Comment:
I think this should be `plugin` not `plugins` based on imp https://github.com/chartjs/Chart.js/blob/2.9/src/core/core.plugins.js#L152
```suggestion
    plugin: Plugin;
```